libs/gundeck-types/src/Gundeck/Types/BulkPush.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE DeriveGeneric              #-}
```

Comment:
Can we make the generated JSON a bit prettier by using sth like https://github.com/wireapp/wire-server/blob/develop/libs/types-common/src/Data/Json/Util.hs#L70

Reply:
will look into it!

Reply:
I think that's all covered by aaf2ea8ece32fb421624f96cd6e04a032d529403?

Reply:
d05faa95 after rebase